nwb_conversion_tools/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py

```diff
@@ -17,28 +24,51 @@ class CellExplorerSortingInterface(BaseSortingExtractorInterface):
     def __init__(self, file_path: FilePathType):
         super().__init__(spikes_matfile_path=file_path)
         self.source_data = dict(file_path=file_path)
+        spikes_matfile_path = Path(file_path)
 
         session_path = Path(file_path).parent
         session_id = session_path.stem
-        spikes_cellinfo_file_path = session_path / f"{session_id}.spikes.cellinfo.mat"
-        if spikes_cellinfo_file_path.is_file():
-            cell_info = loadmat(spikes_cellinfo_file_path).get("spikes", np.empty(0))
-            cell_info_fields = cell_info.dtype.names
+
+        assert spikes_matfile_path.is_file(), f"The spikes_matfile_path ({spikes_matfile_path}) must exist!"
```

Comment:
```suggestion
        assert spikes_matfile_path.is_file(), f"The .spikes.cellinfo.mat file ({spikes_matfile_path}) must exist!"
```

Since the user never actually sees or interacts with a variable by the name of `spikes_matfile_path`, probably best to refer to it in the way they interact with it.

Reply:
@CodyCBakerPhD 
The name of the path argument in the `__init__` function is file_path. What do you think of changing the name from `file_path` to `spikes_matfile_path` which is more indicative?

Reply:
I'm simply talking about the assert message that gets displayed to the user, the internal variable names all look fine

Reply:
All right, I changed it to be a little bit more explicit in the last commit. Marking this a resolved meanwhile.